HelpSource/Guides/Internal-Snooping.schelp

```diff
@@ -57,38 +60,41 @@ Server.filenameSymbol.postln; // the path to the file that defined this class
 
 (
 // print all classes whose names start with 'F'
-Class.allClasses.do({ arg class;
-	if (class.name.asString.at(0) == $F, { class.name.postln; });
-})
+Class.allClasses.do { | class |
+	if (class.name.asString.beginsWith("F")) {
+		class.name.postln
+	}
+}
 )
 
 (
 // find and print all class variable names defined in the system
-Class.allClasses.do({ arg class;
-    if (class.classVarNames.notNil, {
+Class.allClasses.do { | class |
+	if (class.classVarNames.notNil) {
 		// classVarNames is an Array of Symbols
-        class.classVarNames.do({ arg varname;
-            (class.name.asString ++ " " ++ varname.asString).postln;
-        })
-    });
-});
+        class.classVarNames.do { | varname |
+            (class.name.asString ++ " " ++ varname.asString).postln
+        }
+    }
+}
 )
 
 (
 // find and print all methods that contain "ascii"
-Class.allClasses.do({ arg class;
-   	class.methods.do({ arg sel;
+Class.allClasses.do { | class |
+   	class.methods.do { | sel |
    			if(sel.name.asString.find("ascii").notNil) {
-            		(class.name.asString + "-" + sel.name).postln;
+            		(class.name.asString + "-" + sel.name).postln
             }
-      });
-}); ""
+	}
+}
 )
 ::
 
 subsection:: Snooping in Methods
 
 Same thing goes here, if you store things into Methods, you may break something.
```

Comment:
I really don't understand this comment... is it

note::
Even though you may access these data structures, if you store things into them, you may break something.
::